<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera Detection</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string>Camera</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>501</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>History</string>
    </property>
    <widget class="QTableView" name="tableView_Data">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>481</width>
       <height>511</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_videoFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>471</width>
      <height>391</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_customer">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CONFIG CUSTOMER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_employer">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>470</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CONFIG EMPLOYER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_item">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>510</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CONFIG ITEM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_invoice">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>510</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CONFIG INVOICE</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1030</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFime">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionConfig_Customer"/>
    <addaction name="actionConfig_Employee"/>
    <addaction name="actionConfig_Item"/>
    <addaction name="actionConfig_Invoice"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFime"/>
   <addaction name="menuSetting"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionConfig_Customer">
   <property name="text">
    <string>Config Customer</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionConfig_Item">
   <property name="text">
    <string>Config Item</string>
   </property>
  </action>
  <action name="actionConfig_Invoice">
   <property name="text">
    <string>Config Invoice</string>
   </property>
  </action>
  <action name="actionConfig_Employee">
   <property name="text">
    <string>Config Employee</string>
   </property>
  </action>
  <action name="action_customer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>button_customer</string>
   </property>
   <property name="toolTip">
    <string>button_customer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
